### PR TITLE
Opensrp

### PR DESCRIPTION
**Pull request by keymane**
- OpenSRP docker files

### CODE REVIEW

**Comment by ukanga** [comment]
@keymane can you remove the .DS_Store file, also add it to the `.gitignore` file.

File: opensrp/composed/.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```
Comment (ukanga) [inline opensrp/composed/.env]: I don't this file is supposed to be committed.

File: opensrp/composed/sh/start.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Import data
+
+/usr/local/bin/wait-for-it.sh mysql:3306
+
+mysql -u root -p"$MYSQL_ROOT_PASSWORD" -h mysql -e "CREATE DATABASE $MYSQL_MOTECH_DATABASE /*\!40100 DEFAULT CHARACTER SET utf8 */;"
```
Comment (ukanga) [inline opensrp/composed/sh/start.sh]: `-h mysql` could you use a variable for the mysql host as well aka `$MYSQL_HOST`, you could set the variable at the start of the script.

**Review by ukanga — CHANGES_REQUESTED** [review]

**Comment by keymane** [comment]
@ukanga I've made the changes requested

**Review by ukanga — APPROVED** [review]

**Pull request merged**